Mathlib/Order/Filter/Germ.lean

```diff
@@ -837,54 +824,41 @@ theorem const_inf [Inf β] (a b : β) : ↑(a ⊓ b) = (↑a ⊓ ↑b : Germ l 
   rfl
 #align filter.germ.const_inf Filter.Germ.const_inf
 
-instance semilatticeSup [SemilatticeSup β] : SemilatticeSup (Germ l β) :=
-  { Germ.partialOrder with
-    sup := (· ⊔ ·)
-    le_sup_left := fun f g =>
-        inductionOn₂ f g fun _f _g => eventually_of_forall fun _x => le_sup_left
-    le_sup_right := fun f g =>
-      inductionOn₂ f g fun _f _g => eventually_of_forall fun _x => le_sup_right
-    sup_le := fun f₁ f₂ g =>
-      inductionOn₃ f₁ f₂ g fun _f₁ _f₂ _g h₁ h₂ => h₂.mp <| h₁.mono fun _x => sup_le }
-
-instance semilatticeInf [SemilatticeInf β] : SemilatticeInf (Germ l β) :=
-  { Germ.partialOrder with
-    inf := (· ⊓ ·)
-    inf_le_left := fun f g =>
-        inductionOn₂ f g fun _f _g => eventually_of_forall fun _x => inf_le_left
-    inf_le_right := fun f g =>
-      inductionOn₂ f g fun _f _g => eventually_of_forall fun _x => inf_le_right
-    le_inf := fun f₁ f₂ g =>
-      inductionOn₃ f₁ f₂ g fun _f₁ _f₂ _g h₁ h₂ => h₂.mp <| h₁.mono fun _x => le_inf }
-
-instance lattice [Lattice β] : Lattice (Germ l β) :=
-  { Germ.semilatticeSup, Germ.semilatticeInf with }
-
-instance distribLattice [DistribLattice β] : DistribLattice (Germ l β) :=
-  { Germ.semilatticeSup, Germ.semilatticeInf with
-    le_sup_inf := fun f g h =>
-      inductionOn₃ f g h fun _f _g _h => eventually_of_forall fun _ => le_sup_inf }
+instance instSemilatticeSup [SemilatticeSup β] : SemilatticeSup (Germ l β) where
+  le_sup_left f g := inductionOn₂ f g fun _f _g => eventually_of_forall fun _x ↦ le_sup_left
+  le_sup_right f g := inductionOn₂ f g fun _f _g ↦ eventually_of_forall fun _x ↦ le_sup_right
+  sup_le f₁ f₂ g := inductionOn₃ f₁ f₂ g fun _f₁ _f₂ _g h₁ h₂ ↦ h₂.mp <| h₁.mono fun _x ↦ sup_le
+
+instance instSemilatticeInf [SemilatticeInf β] : SemilatticeInf (Germ l β) where
+  inf_le_left f g := inductionOn₂ f g fun _f _g ↦ eventually_of_forall fun _x ↦ inf_le_left
+  inf_le_right f g := inductionOn₂ f g fun _f _g ↦ eventually_of_forall fun _x ↦ inf_le_right
+  le_inf f₁ f₂ g := inductionOn₃ f₁ f₂ g fun _f₁ _f₂ _g h₁ h₂ ↦ h₂.mp <| h₁.mono fun _x ↦ le_inf
+
+instance instLattice [Lattice β] : Lattice (Germ l β) where
+  __ := instSemilatticeSup
+  __ := instSemilatticeInf
+
+instance instDistribLattice [DistribLattice β] : DistribLattice (Germ l β) where
+  le_sup_inf f g h := inductionOn₃ f g h fun _f _g _h ↦ eventually_of_forall fun _ ↦ le_sup_inf
 
 @[to_additive]
-instance orderedCommMonoid [OrderedCommMonoid β] : OrderedCommMonoid (Germ l β) :=
-  { Germ.partialOrder, Germ.commMonoid with
```

Comment:
Are we sure this ordering of sources is not intentional?

Reply:
Yes, I rewrote quite a lot of this file before the port. Note that the source do not overlap